Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.yml

```diff
@@ -0,0 +1,218 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: OctoxLabs
+  version: -1
+configuration:
+- defaultvalue: 127.0.0.1
+  display: IP
+  additionalinfo: Your Octox Labs Platform IP
+  name: octox_ip
+  required: true
+  type: 0
+- displaypassword: API Key
+  additionalinfo: Your Octox Labs API Key. (https://github.com/octoxlabs/py-octoxlabs#getting-started)
+  name: octox_token
+  required: true
+  hiddenusername: true
+  type: 9
```

Comment:
Since we're using `type: 9`, to actually access the token, we need to access it using `demisto.params().get('credentials', {}).get('password')` in the code. Take a look at [Authentication Parameter Type](https://xsoar.pan.dev/docs/integrations/parameter-types#authentication).

Reply:
Ok, solved it.
[https://github.com/demisto/content/pull/21175/commits/3395d5953766f6018f163964acf3fe51190c1ea1#diff-8c6efb9d03b6cf13cbf8bb15cfbfec49ee8f5ac38f28ce10be71fd4daf4694acL187](https://github.com/demisto/content/pull/21175/commits/3395d5953766f6018f163964acf3fe51190c1ea1#diff-8c6efb9d03b6cf13cbf8bb15cfbfec49ee8f5ac38f28ce10be71fd4daf4694acL187)